### ISSUE TITLE
What do the following things represent

### ISSUE THREAD

**Issue by SharhadBashar** [visinf/1-stage-wseg#26]
In the step function in triain.py, you return the following things from the network
`cls_out, cls_fg, masks, mask_logits, pseudo_gt, loss_mask = self.enc(image, image_raw, gt_labels)`
What do these things represent:

1. cls_out
2. cls_fg
3. masks
4. mask_logits
5. pseudo_gt
6. moss_mask
